_rules/error-field-is-not-identified-using-color-only-i3z4v8.md

```diff
@@ -0,0 +1,175 @@
+---
+id: i3z4v8
+name: Error field is not identified using color differences only
+rule_type: atomic
+description: |
+  This rule checks that error fields can be identified not only by color differences but through another visual means.
+accessibility_requirements:
+  wcag21:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  -  DOM Tree
+  -  CSS Styling
+acknowledgments:
+  authors:
+    -  Giacomo Petri
+---
+
+## Description
+
+This rule checks that errors are not identified only through color differences.
+
+## Applicability
+
+This rule applies to each [HTML element][] that is [visible][] and has one of the following [semantic roles][]:
+- checkbox,
+- combobox,
+- listbox,
+- menuitemcheckbox,
+- menuitemradio,
+- radio,
+- searchbox,
+- slider,
+- spinbutton,
+- switch or
+- textbox.
+
+## Expectation
+
+Each test target either has no [visible][] [form field error indicators][], or at least one of the [visible][] [form field error indicators][] identifies the presence of errors not only through color differences.
```

Comment:
Seeing a potential pitfall here (also when looking at examples):
This somewhat implicitly consider that we have something to compare the error indicator to, in order to decide whether it is color differences or more (i.e. "differences" with **what**?)

This could either be another field, not in error, in the same form, or the same field when not errored. 
The second solution somewhat requires a statefull rule and the tester to act (and put the field in error). This hopefully will be doable soonish (once we've solved #1953 / #2045), but that may still take time.
The first point could be doable by targeting pairs of field, passing if they are both errored or none errored, and comparing if only one is errored 🤔

Reply:
> This could either be another field, not in error, in the same form
Comparing a pair of inputs, where one fails and the other does not, is not an optimal approach. Additionally, inputs may have different styles, such as checkboxes or text fields. Therefore, it is advisable to only compare inputs of the same type and with the same CSS.

> Put otherwise: if there is only one form field in the page, and it is in error, how do we check the rule?
Limiting the rule to a specific pair of inputs (assuming the previous concern is addressed) would, in my opinion, be too restrictive.

I think we should go with the second option. @Jym77, should we put this rule on hold till https://github.com/act-rules/act-rules.github.io/issues/1953 / https://github.com/act-rules/act-rules.github.io/issues/2045 are solved?

Reply:
I think it's a list worth bringing to attention that this is another rule proposal that would benefit from being statefull, so that we are sure the solution can handle it.